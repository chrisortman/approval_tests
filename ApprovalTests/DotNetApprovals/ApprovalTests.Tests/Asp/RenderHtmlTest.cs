using ApprovalTests.Reporters;
using Asp.Net.Demo.Orders;
using NUnit.Framework;

namespace ApprovalTests.Tests.Asp
{
    [TestFixture]
    [UseReporter(typeof(FileLauncherReporter))]
    public class RenderHtmlTest
    {
        [Test]
				public void TestSimpleInvoice()
        {
					ApprovalTests.Asp.Approvals.ApproveAspPage(new InvoiceView().TestSimpleInvoice);
					//ApprovalTests.Asp.Approvals.ApproveUrl("http://localhost:1359/Orders/InvoiceView.aspx?TestSimpleInvoice");
        }
    }
}
