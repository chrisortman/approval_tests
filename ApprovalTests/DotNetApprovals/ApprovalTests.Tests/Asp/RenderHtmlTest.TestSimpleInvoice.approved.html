

<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">

<html xmlns="http://www.w3.org/1999/xhtml">
<head><base href="http://localhost:1359/Orders"><title>

</title><link rel="stylesheet" type="text/css" href="Styles\Site.css" /></head>
<body>
		<form method="post" action="InvoiceView.aspx?TestSimpleInvoice" id="form1">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTExMTIwODg2Nw9kFgICAw9kFggCBQ8PFgIeBFRleHQFEFNhbW15IFN3ZWV0IFNob3BkZAIJDw8WAh8ABQMxMjNkZAINDw8WAh8ABQkzLzE0LzE1OTJkZAIRDzwrABECAA8WBh4MRGF0YVNvdXJjZUlEZB4LXyFEYXRhQm91bmRnHgtfIUl0ZW1Db3VudAICZAEQFgAWABYAFgJmD2QWBgIBD2QWCGYPDxYCHwAFCkNhbmR5IEJhcnNkZAIBDw8WAh8ABQUkMC41MGRkAgIPDxYCHwAFByQyMDAuMDBkZAIDDw8WAh8ABQckMTAwLjAwZGQCAg9kFghmDw8WAh8ABQxGaXp6eSBTdHJhd3NkZAIBDw8WAh8ABQUkMC43MGRkAgIPDxYCHwAFByQxMDAuMDBkZAIDDw8WAh8ABQYkNzAuMDBkZAIDDw8WAh4HVmlzaWJsZWhkZBgBBQlHcmlkVmlldzEPPCsADAEIAgFkUpuDJ8oNo8Yyim0ey5jhVNpnLMJSzWOhhMwvvh9QYdM=" />

		<div>
		
			<img id="Image1" src="../images/acmelogo.gif" height="25%" width="25%" />
			<br />
			<span id="lblCompanyName">Company Name : </span>
			<span id="lblCompanyNameValue">Sammy Sweet Shop</span>
			<br />
			<br />
			<span id="lblOrderNumber">Order Number : </span>
			<span id="lblOrderNumberValue">123</span>
			<br />
			<br />
			<span id="lblOrderDate">Order Date : </span>
			<span id="lblOrderDateValue">3/14/1592</span>
			<br />
			
			<br />
			<div>
	<table cellspacing="0" rules="all" border="1" id="GridView1">
		<tr>
			<th scope="col">ItemName</th><th scope="col">ItemPrice</th><th scope="col">ItemQuantity</th><th scope="col">SubTotal</th>
		</tr><tr>
			<td>Candy Bars</td><td>$0.50</td><td>$200.00</td><td>$100.00</td>
		</tr><tr>
			<td>Fizzy Straws</td><td>$0.70</td><td>$100.00</td><td>$70.00</td>
		</tr>
	</table>
</div>
		
		</div>
		`</form>
</body>
</html>
